### ISSUE TITLE
Make Layout/EmptyLineAfterGuardClause accept RuboCop directive after guard clause

### ISSUE THREAD

**Issue by Drenmi** [rubocop-hq/rubocop#9146]
## Expected behavior

I expected the cop to not flag the case where the line after the guard clause is a RuboCop directive, e.g.:

```ruby
# rubocop:disable Metrics/LineLength
return if ...
return if ...
return if ...
# rubocop:enable Metrics/LineLength
```

## Actual behavior

RuboCop registers an offense.

## Steps to reproduce the problem

Run RuboCop on the following:

```ruby
# rubocop:disable Metrics/LineLength
return error(:validation_error, "Some very descriptive error message") unless everything_is_free_of_errors?
# rubocop:enable Metrics/LineLength
```

## RuboCop version

```
1.3.1 (using Parser 2.7.2.0, rubocop-ast 1.1.1, running on ruby 2.7.1 x86_64-darwin18)
```

**Comment by tejasbubane** [comment]
@Drenmi I was not able to reproduce this with latest rubocop (1.5.1)

```
$ cat test.rb                                            
# frozen_string_literal: true

def foo
  # rubocop:disable Layout/LineLength
  return error(:validation_error, 'Some very very very very very loooooooooooooong descriptive error message') unless everything_is_free_of_errors?
  return error(:validation_error, 'Some very very very very very loooooooooooooong descriptive error message') unless everything_is_free_of_errors?
  # rubocop:enable Layout/LineLength
end
```

```
$ rubocop --only Layout/EmptyLineAfterGuardClause test.rb
Inspecting 1 file
.

1 file inspected, no offenses detected
```

```
$ rubocop -V       
1.5.1 (using Parser 2.7.2.0, rubocop-ast 1.3.0, running on ruby 2.7.2 x86_64-linux)
```
